app/import.go

```diff
@@ -167,9 +167,5 @@ func (a *App) finalizeImport(dryRun bool) *model.AppError {
 	if dryRun {
```

Comment:
I can remove now the entire finalizeImport machinery, but I'm not sure if is a good idea to maintain it for future use cases (we can read it again later, of course)

Reply:
0/5 happy to leave this in.